src/Compilers/VisualBasic/Test/Syntax/TestSyntaxNodes.vb

```diff
@@ -2796,6 +2796,25 @@ End Module
             Assert.Throws(Of ArgumentNullException)(Sub() newTree.GetChangedSpans(BlankTree))
         End Sub
 
+        <Fact>
+        Public Sub TestSyntaxTree_GetChangesWhenSubChangedToFunction()
+            Dim parseLines = 
+                Function(lines As String()) SyntaxFactory.ParseSyntaxTree(String.Join(vbCrLf, lines))
+
+            Dim oldTree = parseLines({ "Module M", "  Sub M()", "    Console.WriteLine(42)", "  End Sub", "End Module"})
+            Dim newTree = parseLines({ "Module M", "  Async Function M() of Task", "    Console.WriteLine(42)", "  End Function", "End Module"})
```

Comment:
When I used verbatim strings for code, this test failed on Unix, because of different line endings, which in turn mean different `TextSpan`s.

So I switched to this approach, which ensures `\r\n` is used on any OS. Is there a better way to do this?

Reply:
Did the pattern used by other `GetChanges` tests (such as `TestSyntaxTree_GetChangesInValid`) cause the same problem?

---
In reply to: [226878038](https://github.com/dotnet/roslyn/pull/30634#discussion_r226878038) [](ancestors = 226878038)

Reply:
No, because they're not comparing a multi-line string against a constant `TextSpan`. So this seems to be the only test which has this problem.